lib/utilities/will-interrupt-process.js

```diff
@@ -5,13 +5,39 @@
 // When the process interrupted("CTR+C") all the registered handlers are being invoked via `capture-exit`.
 // The process finishes after the last interruption handler is settled.
 
-let exit = require('capture-exit');
-exit.captureExit();
+let captureExit = require('capture-exit');
 
-let windowsCtrlCTrap;
-let handlers = [];
+let _process,
+    windowsCtrlCTrap,
+    originalIsRaw,
+    handlers = [];
 
 module.exports = {
+  capture(outerProcess) {
+    if (_process) {
+      throw new Error('process already captured');
+    }
+
+    _process = outerProcess || process;
```

Comment:
Originally I've added this fallback for convenience in unit tests [where actual `process` doesn't matter](https://github.com/ember-cli/ember-cli/pull/7086/files#diff-b206c498c137f28dedad7f205bfe68e6R60). But agree it's better to simplify here and throw if anything strange happens.